File: API/Core/Entities/StavkaPorudzbine.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Core.Entities
{
   
    public class StavkaPorudzbine : BaseEntity
    {
        [Key]
        public Guid StavkaPorudzbineId { get; set; }
        public int BrojStavki { get; set; }
        public Porudzbina Porudzbina { get; set; }
        public Guid PorudzbinaId { get; set; }
        public VelicinaDresa VelicinaDresa { get; set; }
        public Guid VelicinaDresaId { get; set; }
    }
}

File: Frontend/ClientApp/src/app/shared/models/stavka-porudzbine.ts
import { Porudzbina } from "./porudzbina";
import { VelicinaDresa } from "./velicina-dresa";

export class StavkaPorudzbine {
    stavkaPorudzbineId: string;
    brojStavki: number;
    porudzbina: Porudzbina;
    porudzbinaId: string;
    velicinaDresa: VelicinaDresa;
    velicinaDresaId: string;

    constructor(
        stavkaPorudzbineId: string,
        brojStavki: number,
        porudzbina: Porudzbina,
        porudzbinaId: string,
        velicinaDresa: VelicinaDresa,
        velicinaDresaId: string
    ) {
        this.stavkaPorudzbineId = stavkaPorudzbineId;
        this.brojStavki = brojStavki;
        this.porudzbina = porudzbina;
        this.porudzbinaId = porudzbinaId;
        this.velicinaDresa = velicinaDresa;
        this.velicinaDresaId = velicinaDresaId;
    }
}
    export class StavkaPorudzbineCreation {
        brojStavki: number;
        porudzbinaId: string | null;
        velicinaDresaId: string;
    
        constructor(
            brojStavki: number,
            porudzbinaId: string,
            velicinaDresaId: string
        ) {
            this.brojStavki = brojStavki;
            this.porudzbinaId = porudzbinaId;
            this.velicinaDresaId = velicinaDresaId;
        }
    
}

export class StavkaPorudzbineUpdate {
    stavkaPorudzbineId: string;
    brojStavki: number;
    

    constructor(
        stavkaPorudzbineId: string,
        brojStavki: number

       
        
    ) {
        this.stavkaPorudzbineId=stavkaPorudzbineId
        this.brojStavki = brojStavki;
        
    }

}
